app/update.go

```diff
@@ -82,22 +85,78 @@ func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		var cmd tea.Cmd
 		cmd = m.systemInfo.Update(msg)
 		return m, cmd
-	
+
 	case systeminfoview.SlowStatusMsg:
 		var cmd tea.Cmd
 		cmd = m.systemInfo.Update(msg)
 		return m, cmd
-	
+
 	case systeminfoview.TickMsg:
 		var cmd tea.Cmd
 		cmd = m.systemInfo.Update(msg)
 		return m, cmd
-	
+
 	case systeminfoview.SpinnerTickMsg:
 		var cmd tea.Cmd
 		cmd = m.systemInfo.Update(msg)
 		return m, cmd
 
+	case nodesview.Msg:
```

Comment:
We just have the delegate to current view function. No need to do it like this:

https://github.com/mosonyi/swarmcli/blob/28fea4665a033e330596c21b5ffe951f437f77cf/app/update.go#L166